src/renderer/dx/DxRenderer.cpp

```diff
@@ -193,6 +193,7 @@ DxEngine::~DxEngine()
         SwapChainDesc.SampleDesc.Count = 1;
         SwapChainDesc.AlphaMode = DXGI_ALPHA_MODE_UNSPECIFIED;
 
+#ifdef EXTERNAL_BUILD

```

Comment:
I don't think you need this anymore with the manifest change that was just committed... right?

Reply:
I used the Resource Hacker to view the application manifest file for C:\Windows\System32\conhost.exe and found that there is no supportedOS setting, so the conhost.exe built inside Windows is correctly set to DXGI_SCALING_NONE .  

What is the difference between DXGI_SCALING_STRETCH and DXGI_SCALING_NONE? @dgriffen is needed to answer, thank you.

Reply:
It's a bit more complicated than that. The binaries in the OS don't _need_ `supportedOS` in their manifests, so it's actually setting `DXGI_SCALING_STRETCH`. I know that's unusual.

Reply:
`DXGI_SCALING_STRETCH` is the only option that is allowable on windows 7. The difference only matters if the backing buffer is not equal in size to the presentation target. In the case of the DX renderer for the terminal I don't think this is ever the case so there will be equivalent behavior with either setting.